templates/pools.html

```diff
@@ -0,0 +1,577 @@
+                                transform="translate(-96.48 -21)" fill="#585268" />
+                        </svg>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <br />
+        <div>Staking pool deposit adresses are based on on-chain data. Please open an <a
+                href="https://github.com/gobitfly/eth2-beaconchain-explorer/issues">issue</a> if you want to add an
+            Ethereum staking pool.</div>
+        <div class="row mt-4">
+            <div class="col-md-12">
+                <div id="poolTable" class="table-responsive card px-0 pb-1 pt-2 mb-2">
+                    <table class="table" id="staking-pool-table">
+                        <thead>
+                            <tr>
+                                <th>Pool</th>
+                                <th>Validators</th>
+                                <th>Avg. Performance 31d</th>
+                                <th>Avg. Performance 7d</th>
+                                <th>Avg. Performance 1d</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {{range .Data.PoolInfos}}
+                            <tr>
+                                <td>{{.Name}}</td>
+                                <td>{{.Count}}</td>
+                                <td>{{formatBalanceChange .AvgPerformance31d $.Currency}}</td>
+                                <td>{{formatBalanceChange .AvgPerformance7d $.Currency}}</td>
+                                <td>{{formatBalanceChange .AvgPerformance1d $.Currency}}</td>
+                            </tr>
+                            {{end}}
+                        </tbody>
+                    </table>
+                </div>
+                <span class="px-2" id="lastUpdate"></span>
+            </div>
+        </div>
+    </div>
+</section>
+<div id="tempcont" class=""></div>
+<script src="/js/jquery.min.js"></script>
+<script src="/js/luxon.min.js"></script>
+<script>
+
+    window.addEventListener('load', function () {
+        Highcharts.chart('pools-distribution-chart', {
```

Comment:
This can stay here, but calling a function from the JS block and passing the golang object would be a bit cleaner